static/app/views/performance/landing/widgets/widgets/performanceScoreWidget.tsx

```diff
@@ -44,17 +45,16 @@ export function PerformanceScoreWidget(props: PerformanceWidgetProps) {
       }
     : undefined;
 
+  const moduleURL = useModuleURL('vital');
```

Comment:
Ignore, this would be way to much to change rn lol, but I feel like this module should be called `web-vitals` in code.

Reply:
Oh yeah, definitely. Also some of these use underscores vs. dashes and so on 😭